weather_dl/download_pipeline/pipeline_test.py

```diff
@@ -632,5 +635,19 @@ def test_target_name_date_and_target_directory_additional_partitions_in_path(sel
         self.assertEqual(target_name, "somewhere/expver-1/2017/01/15-pressure-500.nc")
 
 
+class ConfigTest(unittest.TestCase):
+
+    def setUp(self):
+        self._data_dir = f'{next(iter(weather_dl.__path__))}/../configs'
```

Comment:
I feel like this could be cleaner but it was a bit fiddly getting the paths working both locally (Windows) & in CI. Used `weather_sp/test_data` as an example here but happy to change if there are any suggestions!

Reply:
Yeah, I agree with you. The example in `weather_sp/test_data` is not the cleanest. I'm ok to merge this from now and address this path string all at once later.

Reply:
nit: I see no need for data_dir to be private in these tests.